web/src/components/SampleBrowser.vue

```diff
@@ -7,14 +7,23 @@ export default {
     return {
       loading: true,
       samples: [],
+      error: false,
     };
   },
   computed: {
 
   },
   async created() {
     this.loading = true;
-    this.samples = (await SampleService.list()).data;
+
+    try {
+      const { data } = await SampleService.list();
+      this.samples = data;
+    } catch (err) {
+      this.samples = [];
```

Comment:
superfluous, I think.  `this.samples` can only ever be `[]` at the time this line runs.

Reply:
```suggestion
```